home_works/lesson15/human.py

```diff
@@ -0,0 +1,47 @@
+from random import randint
+
+from home_works.lesson15.bank_account import BankAccount
+from home_works.lesson15.house import House
+from home_works.lesson15.car import Car
+from home_works.lesson15.purchse_permit import house_purchase_permit, car_purchase_permit
+
+
+class Human:
+    created_passport_nums = set()
+
+    def __init__(self, name, passport_id):
+        if passport_id in self.created_passport_nums:
+            raise ValueError('...')
+
+        self.name = name
+        self.passport_id = passport_id
+        self.__bank_account = BankAccount(self.passport_id)
+        self.house = None
+        self.car = None
+
+    def __repr__(self):
+        return f'{self.name, self.passport_id}'
+
+    def work(self):
+        salary = randint(5, 11)
+        self.__bank_account.increase_money(salary)
+
+    def buy(self, choice):
+        if isinstance(choice, House):
```

Comment:
better to use only one method to check all props.. call it like `permissions_property`. Pass there prop and there check instances and define what do we need to do..